### PR TITLE
Add white background to cards by default

### LINKED ISSUE #17117: v4 - Card missing white background or bg-default missing
Cards background are by default transparent. But I need it to be white (like formaly panel-default).

So we need `card-default` or `bg-default` to fix this.

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #17117

### CODE REVIEW

**Comment by zacechola** [comment]
I'm fine with the background color, though probably use lowercase for the hex value. My only issue with naming it "white" or similar is that it could be (should be?) a variable value.

**Comment by kkirsche** [comment]
ah ok. We can certainly use lowercase hex codes @zacechola — I've pushed an update addressing that.

Honestly, the reason I didn't make it a variable value is to avoid stepping on #17069

**Comment by mdo** [comment]
Let's nix the `card-white` variation and go with a default background on `.card` instead.

**Comment by kkirsche** [comment]
OK.

**Comment by kkirsche** [comment]
@mdo updated to include it on `.card`

**Comment by cvrebert** [comment]
@kkirsche The commit message doesn't make sense anymore

**Comment by kkirsche** [comment]
@cvrebert — Changed. Figured you guys would have changed it when merging honestly to be what you wanted but no big deal

File: scss/_card.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@ $card-border-radius-inner: ($card-border-radius - $card-border-width);
 $card-cap-bg: #f5f5f5;
 
 .card {
+  background-color: #fff;
```
Comment (mdo) [inline scss/_card.scss]: Placement of this property is incorrect—please place it before `border`. See the property order in our Sass lint config for more details.

Comment (kkirsche) [inline scss/_card.scss]: Changed. Is there a reason for the (arguably) odd order of values instead of just using alphabetical which is clear?

Comment (cvrebert) [inline scss/_card.scss]: Yes:
http://codeguide.co/#css-declaration-order
http://markdotto.com/2011/11/29/css-property-order/

**Comment by mdo** [comment]
Thanks for all the back and forth! <3

**Comment by kkirsche** [comment]
Anytime @mdo — just trying to help out :+1:

**Pull request merged**